modules/data-loading/pages/kafka-loader-user-guide.adoc

```diff
@@ -1,25 +1,112 @@
 = Kafka Loader User Guide
+:description: This guide walks you through how to use the Kafka Loader to load data from an external Kafka cluster into your TigerGraph instance.
 
 == Overview
 
 Kafka is a popular pub-sub system in enterprise IT, offering a distributed and fault-tolerant real-time data pipeline. TigerGraph's Kafka Loader feature lets you easily integrate with a Kafka cluster and speed up your real-time data ingestion. It is easily extensible using the many plugins available in the Kafka ecosystem.
 
 The Kafka Loader consumes data in a Kafka cluster and loads data into the TigerGraph system.
 
-== Architecture 
+== Architecture
 
 From a high level, a user provides instructions to the TigerGraph system through GSQL, and the external Kafka cluster loads data into TigerGraph's RESTPP server. The following diagram demonstrates the Kafka Loader data architecture.
 
 image::kafka-loading-architecture.png[Kafka Data Loader Architecture]
 
 == Prerequisites
 
-You should have a Kafka cluster configured and set up in your environment.
+* You should have a Kafka cluster configured and set up in your environment.
 
-Once you have the external Kafka cluster setup, you need to prepare the following two configuration files and place them in your desired location in TigerGraph system:
+* Once you have the external Kafka cluster setup, you need to prepare the following two configuration files and place them in your desired location in TigerGraph system:
+** <<_kafka_data_source_configuration_file>>
+** <<_kafka_topic_and_partition_configuration_file>>
 
-. Kafka data source configuration file: This file includes the external Kafka broker's domain name and port. Through the configuration file, TigerGraph system knows the location and port of the external Kafka broker.  Please see an example in link:#_1_define_the_data_source[*Step 1. Define the Data Source*].
-. Kafka topic and partition configuration file: This file includes the Kafka topic, partition list, and start offset for the loading messages.  Please see an example in link:#_2_create_a_loading_job[*Step 2. Create a Loading Job*].
+[#_kafka_data_source_configuration_file]
+=== Kafka data source configuration file
+
+The data source configuration file's content, structured as a JSON object, describes the Kafka server's global settings, including the data source IP and port.
+A sample `kafka.conf` is shown in the following example:
+
+[source,javascript]
+----
+{
+  "broker": "192.168.1.11:9092",
+}
+----
+
+The `broker` key is required, and its value is a fully qualified domain name (or IP) and port.
+Additional Kafka configuration parameters may be provided (see Kafka documentation) by using the optional `"kafka_config"` key.
```

Comment:
Where should the user look? Should we provide a link?
This makes me think that in the intro, we perhaps should be more explicit about what we mean by Kafka.
* Kafka gained fame as an open-source Apache project: https://kafka.apache.org/
* Many Enterprise users use Confluent Kafka (https://www.confluent.io/lp/apache-kafka). 
Do we support both?

Reply:
Thanks for the comments. I actually didn't touch this page much in this ticket, and some of the edits were made because of merge conflicts that happened when I tried to merge the PR. 

I think we support both versions of Kafka but I'm not sure. I'll confirm with the engineer.